Feature/NewBookingFromTopMenu.feature

```diff
@@ -0,0 +1,31 @@
+Feature: New booking from the home page top menu
```

Comment:
change the feature name to just New booking

Reply:
Done